src/helpers/contactHelpers.ts

```diff
@@ -0,0 +1,199 @@
+import { Contact, InvitedContact, CodeReceivedContact, AcceptedContact, MutualContact } from '../models/Contact';
```

Comment:
This kind of helper functionality is very feature specific,  I think this should go under `src/contact`. This is also true for some already existing code in the `helpers` dir.

Reply:
You want to create a new folder to contain one file? Can you elaborate a bit on this?